test/testcafe/spec/v1/MfaOnlySmsChallenge_spec.js

```diff
@@ -0,0 +1,83 @@
+import { ClientFunction, RequestMock } from 'testcafe';
+import MfaChallengePageObject from '../../framework/page-objects-v1/MfaChallengePageObject';
+import mfaChallengeResponse from '../../../../playground/mocks/data/mfa-only/factor-challenge-sms.json';
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/api/v1/authn/introspect')
+  .respond(mfaChallengeResponse);
+
+
+fixture('Mfa-only SMS challenge');
+
+const renderWidget = ClientFunction((settings) => {
+  window.renderPlaygroundWidget(settings);
+});
+
+async function setup(t, options = {}) {
+  const pageObject = new MfaChallengePageObject(t);
+  await pageObject.navigateToPage({ render: true });
+
+  // Render the widget for interaction code flow
+  await pageObject.mockCrypto();
+  await renderWidget({
+    stateToken: '00EPMl_-Vn1hX-3CRz8qjcmdHIIXx89qaNR6d705mU',
+    status: 'mfa_required',  
+    features: {
+      hideSignOutLinkInMFA: true
+    },
+    ...options
+  });
+
+  return pageObject;
+}
+
+const stupportedLanguages = [
+  'en',
+  'cs',
+  'da',
+  'de',
+  'el',
+  'es',
+  'fi',
+  'fr',
+  'hu',
+  'id',
+  'in',
+  'it',
+  'ja',
+  'ko',
+  'ms',
+  'nb',
+  'nl-NL',
+  'ok-PL',
+  'pl',
+  'pt-BR',
+  'ro',
+  'ru',
+  'sv',
+  'th',
+  'tr',
+  'uk',
+  'vi',
+  'zh-CN',
+  'zh-TW',
+];
+
+stupportedLanguages.forEach( (lang) => {
+  test
+    .requestHooks(mock)('Show SMS challenge for MFA only setup "' + lang + '" locale', async t => {
+      const pageObject = await setup(t, {
+        language: lang
+      });
+
+      // wait for all elements to render
+      await t.wait(5000);      
```

Comment:
This test will take a long time given it waits 5 secs and we are testing all languages.
- Do we really need to test all languages? I don't see any difference in the code in this PR for one lang vs another
- Can we get away with a unit test? This is v1, and we have a few specs for mfa required and challenge

Reply:
This is demo that changing the language do not overlap the elements ... It is not related to specific language but for language transition. I can reduce the number leave just 2 languages  ...

Reply:
Got it. If we can, lets run 3-5 languages then

Reply:
Also, what I was asking about unit tests, I see we have a bunch of MFA_REQUIRED mocks for unit tests here: https://github.com/okta/okta-signin-widget/tree/master/test/unit/helpers/xhr, any chance we can use those? Not a big deal adding another mock, but if we have one already let's reuse it (Might not work for an e2e test, but it will for a unit test)

Reply:
This is mocking the response from ADFS challenge. I am not sure any of the existing mock will be similar